apps/aecore/src/aecore.app.src

```diff
@@ -34,8 +34,7 @@
     idna,
     inets,
     nat,
-    aecuckoo,
-    aecuckooprebuilt,
```

Comment:
Please confirm validity of docs.

I see aecuckooprebuilt and aecuckoo at least in docs/configuration.md and docs/cuda-miner.md

Reply:
Please also check https://github.com/aeternity/aeternity/wiki/Troubleshooting

Reply:
The docs should stay unchanged, configuration of the miners is still part of the ae node: https://github.com/aeternity/aeternity/blob/aec_pow_cuckoo-refactor/apps/aecore/src/aec_mining.erl

Reply:
Could you please confirm.

At least in cuda-miner.md I see aecuckoo and aecuckooprebuilt used as file/directory path. Could you please confirm those still valid.

I did not check Troubleshooting wiki page in detail, it would be nice if you could confirm.

Reply:
There is at least this line:
```
cd apps/aecuckoo && make cuda29
```
which is not valid anymore, there is no `apps/aecuckoo`, `aeckuckoo` is in a separate repository.

Reply:
I won't be able to update the docs for this PR, I created a ticket for that https://www.pivotaltracker.com/n/projects/2124891/stories/163979583